.github/scripts/fxcore-sync-up-version.js

```diff
@@ -1,44 +1,63 @@
-const templateConfig = path.join(
+const templateConfigFile = path.join(
   fxCorePath,
   "./src/common/templates-config.json"
 );
-const templateConfigFile = fse.readJsonSync(templateConfig);
+const templateConfigs = JSON.parse(fs.readFileSync(templateConfigFile));
+const command = process.argv[2];
 
-console.log(
-  `================== template version in fx-core configurate as ${templateConfigFile.version} ==================`
-);
-if (!semver.prerelease(templateVersion)) {
-  if (!semver.intersects(templateConfigFile.version, templateVersion)) {
-    const parsedTemplateVersion = semver.parse(templateVersion);
-    const templateVersionRange = `~${parsedTemplateVersion.major}.${parsedTemplateVersion.minor}`;
-    console.log(
-      `================== template config version is not match with template latest release version, need bump up config version ${templateVersionRange} ==================`
-    );
-    templateConfigFile.version = templateVersionRange;
-  }
-  templateConfigFile.useLocalTemplate = false;
-} else if (templateVersion.includes("rc")) {
-  console.log("sync up template in fx-core as 0.0.0-rc");
-  templateConfigFile.version = "0.0.0-rc";
-  templateConfigFile.useLocalTemplate = false;
+if (command === 'syncVersion') {
+  syncTemplateVersion(templateVersion, templateConfigs);
+} else if (command === 'updateUseLocalFlag') {
+  updateUseLocalFlag(templateVersion, templateConfigs)
 } else {
-  console.log("configure fx-core useLocalTemplate as true");
-  templateConfigFile.useLocalTemplate = true;
+  console.warn(
+    `================== invalid command '${command}' for fxcore-sync-up-version.js ==================`
+  )
 }
 
-fse.writeFileSync(templateConfig, JSON.stringify(templateConfigFile, null, 4));
+fs.writeFileSync(templateConfigFile, JSON.stringify(templateConfigs, null, 4));
+
+function syncTemplateVersion(templateVersion, templateConfigs) {
+  console.log(
+    `================== template version: ${templateVersion} ==================`
+  );
+
+  console.log(
+    `================== template version in fx-core configurate as ${templateConfigs.version} ==================`
+  );
+  templateConfigs.localVersion = templateVersion;
+  if (!semver.prerelease(templateVersion)) {
+    if (!semver.intersects(templateConfigs.version, templateVersion)) {
+      const parsedTemplateVersion = semver.parse(templateVersion);
+      const templateVersionRange = `~${parsedTemplateVersion.major}.${parsedTemplateVersion.minor}`;
+      console.log(
+        `================== template config version is not match with template latest release version, need bump up config version ${templateVersionRange} ==================`
+      );
+      templateConfigs.version = templateVersionRange;
+    }
+  } else if (templateVersion.includes("rc")) {
+    console.log("sync up template in fx-core as 0.0.0-rc");
+    templateConfigs.version = "0.0.0-rc";
+  }
+}
+
+function updateUseLocalFlag(templateVersion, templateConfigs) {
```

Comment:
According to new release principle, do we still need useLocal flag? Let me offline get more details from you.